index.html

```diff
@@ -425,7 +425,7 @@ <h5 lang="en">Change language</h5>
         </div>
         <div class="clearfix"></div>
       </div>
-      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved.</span></p>
+      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved. </span><span lang="de"><a href="https://de.opensuse.org/Impressum">Impressum</a>.</span></p>
```

Comment:
Github just removed the html - I meant to say I don't know what span lang=en does.